themes-default/slim/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+        "allowJs": true,
+        "noImplicitAny": true
+    },
+    "include": [
+        "./static/js/*.js"
```

Comment:
This isn't including any subdirs?

Reply:
That's your code, What's this even for?

Reply:
Oh shoot, it is.

It's for enabling typescript typechecking and intellisense in vscode. That allows us to use the jsdoc syntax to add types for all of our functions, etc. and then it'll use those to create our type file so we don't need todo it ourselves.